modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaHelidonServerCodegen.java

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright 2022 OpenAPI-Generator Contributors (https://openapi-generator.tech)
- * Copyright (c) 2022 Oracle and/or its affiliates
+ * Copyright 2022, 2024 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright (c) 2022, 2024 Oracle and/or its affiliates
```

Comment:
for the new java files (not auto-generated) added in this PR, can we use the above instead?

Reply:
I think there is only one new `.java` file in the PR, and I've fixed that copyright notice.

I also fixed one modified `.java` file where I neglected to updated the copyright.

I think those are the only copyright problems, but please let me know if there are more.